tests/validations/mta/mtaValidations.spec.ts

```diff
@@ -94,28 +103,72 @@ describe("mtaValidations", () => {
       await testUpdateMtaDiagnostics([]);
     });
 
+    it("returns no diagnostics when mta.yaml does not exist and clear the current diagnostics", async () => {
+      const expectedDiagnostics: [Uri, Diagnostic[] | undefined][] = [
+        [
+          // when clearing the diagnostic, the diagnostic remains in the cache with "undefined"
```

Comment:
This is more accurate:
```suggestion
          // when clearing the diagnostic, we add an entry with "undefined" to the collection
```
In the real vscode implementation you won't always have the `undefined` after you call `set` (all the collection entries for the same URI are merged)